config.lua

```diff
@@ -49,3 +49,5 @@ Config.ATM_Models = {
     "prop_atm_03",
     "prop_fleeca_atm",
 }
+
+Config.OpenAtmOnlyWhenClose = false -- Open the ATM only when the player is close enought
```

Comment:
I don't see the use case for this being a configuration but rather we want to prevent this always.

Reply:
Not Everyone wants everything, thats why i put the config option.

Reply:
I feel we should just be assuming people can't open ATMs when they aren't at an ATM. Additionally, the `ps-banking:client:open:bank` and `ps-banking:client:open:atm` should probably be turned into functions to prevent the outside triggering of those events. If we want to give people the ability to open banks or atms from elsewhere, they should be exports.

Reply:
i undesrtand but this can be improved, it was only an idea that i have, and im not on my machine i wrote that code on VSWeb and i didnt tested, so if this work i can refractor the code